applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -26,6 +26,10 @@ Configuration
 The module is enabled by selecting :ref:`CONFIG_DESKTOP_USB_ENABLE <config_desktop_app_options>` option.
 The option selects :kconfig:option:`CONFIG_USB_DEVICE_STACK` and :kconfig:option:`CONFIG_USB_DEVICE_HID` Kconfig options.
 
+The :ref:`CONFIG_DESKTOP_USB_ENABLE <config_desktop_app_options>` option is implied by :ref:`CONFIG_DESKTOP_ROLE_HID_DONGLE <config_desktop_app_options>` option.
+The module is enabled by default for nRF Desktop dongles, because the dongles forward HID data to the host connected over USB.
```

Comment:
```suggestion
The module is enabled by default for the nRF Desktop dongles because the dongles forward the HID data to the host connected over USB.
```